data/content_scripts/posting_process/sendkeys.js

```diff
@@ -0,0 +1,766 @@
+// Cross-broswer implementation of text ranges and selections
+// documentation: http://bililite.com/blog/2011/01/17/cross-browser-text-ranges-and-selections/
+// Version: 2.6
+// Copyright (c) 2013 Daniel Wachsstock
+// MIT license:
```

Comment:
This library is still useful for the posting process. The problems that it solves - 
* Inserting text into DOM node.
* Extracting the text from the DOM node. Chrome API provides the text for the context selection easily using the `info` object in the context menu callback. For Firefox, I can only use [Selection](https://developer.mozilla.org/en-US/docs/Web/API/Window/getSelection) but there are some issues with [textareas](http://stackoverflow.com/questions/20419515/window-getselection-of-textarea-not-working-in-firefox). The library takes good care of this as well.

However, it is injected in the same way as `privly.js`. But I can trim some parts of the library, most of the cross browser stuff isn't needed and I can keep only those functions that are needed.